content/en/blog/2024/steering-results/index.md

```diff
@@ -0,0 +1,93 @@
+---
+
+The Istio Steering Committee oversees the administrative aspects of the project, including governance, branding, marketing, and working with the CNCF.
+
+Every year, the leaders in the Istio project [estimate the proportion](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md) of the [hundreds of companies that have contributed to Istio in the past year](https://istio.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=contributions), and uses that metric to proportionally allocate nine Contribution Seats on our Steering Committee.
+
+Then, four Community Seats are voted for by our project members, with candidates being from companies that did not receive Contribution Seats.
+
+We are pleased to share the result of this year's calculation, and changes to our Community Seat holders as a result.
+
+## Contribution seats
+
+The calculation for the 2024-2025 term brings the most diverse set of company representation ever in the Contribution Seats, with five companies[^ibm-red-hat] represented:
+
+{{< centered_block >}}
+<table style="display: table">
+    <thead>
+        <tr>
+            <th>Company</th>
+            <th>Seat allocation</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Google</td>
+            <td>3</td>
+        </tr>
+        <tr>
+            <td>Solo.io</td>
+            <td>2</td>
+        </tr>
+        <tr>
+            <td>IBM/Red Hat</td>
+            <td>2</td>
+        </tr>
+        <tr>
+            <td>DaoCloud</td>
+            <td>1</td>
+        </tr>
+        <tr>
+            <td>Huawei</td>
+            <td>1</td>
+        </tr>
+    </tbody>
+</table>
+{{< /centered_block >}}
+
+The full allocation can be seen in our [formula spreadsheet](https://docs.google.com/spreadsheets/d/1OIwf11xdL3VHi18uhOnHMwQnlXBS22vvhL0m1qraIds/edit#gid=1365082320).
+
+## Community seats
+
+As a result of this year's calculation, two of our Community Seat holders move to Contribution Seats. This creates two extra vacancies, which are allocated to the runners-up of our last election[^daocloud].
+
+We are pleased to welcome our two newest Community Seat holders, Mitch Connors from Aviatrix and Keith Mattix from Microsoft. Both are highly active maintainers and leaders in the project, and we are delighted to have them join the Steering Committee.
+
+## Proposed changes to election timing
+
+Our charter currently allocates Contribution Seats in February and holds the Community Seat election in July. We previously [anticipated a situation](https://github.com/istio/community/pull/609#issuecomment-877302152) where people would change seat type mid-term, and this has now come to pass.
+
+We will therefore be voting on a change to our Charter which will move our Community Seat elections to February, to be held immediately after the allocation of Community Seats. It is our intention that the next annual election be held in February 2025.
```

Comment:
This should say "to be held immediately after the allocation of _**Contribution**_ Seats" (emphasis mine)

Reply:
Good catch, i put these in occasionally to see if anyone is reading closely 😁